inst/tests/tests.Rraw

```diff
@@ -5547,12 +5547,12 @@ test(1375.3, DT[,mean(Petal.Width),by=Species][V1>1,Species:=toupper(Species)]$S
 # Secondary keys a.k.a indexes ...
 DT = data.table(a=1:10,b=10:1)
 test(1376.1, indices(DT), NULL)
-test(1376.2, DT[b==7L,verbose=TRUE], DT[4L], output="Creating new index 'b'")
+test(1376.2, DT[b==7L,verbose=TRUE], DT[4L], output="Creating new index '__b'")
```

Comment:
I am not sure about the naming you used.  I would prefer the old output. `__b` is just internal attribute name for `b` index`.

Reply:
Fine for me. I will adjust so that the messsage completely reflects the return value for the `indices` function:
`b`, but `b__c` for multi-column indices, OK?